
[2016-09-18 22:18:44] request.INFO: Matched route "{route}". {"route":"lote_index","route_parameters":{"_controller":"AppBundle\\Controller\\LoteController::indexAction","_route":"lote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/lote/","method":"GET"} []
[2016-09-18 22:18:44] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-18 22:18:44] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Property AppBundle\Entity\Empresa::$ceIdMatriz does not exist" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\doctrine\common\lib\Doctrine\Common\Persistence\Mapping\RuntimeReflectionService.php line 80 {"exception":"[object] (ReflectionException(code: 0): Property AppBundle\\Entity\\Empresa::$ceIdMatriz does not exist at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\doctrine\\common\\lib\\Doctrine\\Common\\Persistence\\Mapping\\RuntimeReflectionService.php:80)"} []
[2016-09-18 22:26:34] request.INFO: Matched route "{route}". {"route":"lote_index","route_parameters":{"_controller":"AppBundle\\Controller\\LoteController::indexAction","_route":"lote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/lote/","method":"GET"} []
[2016-09-18 22:26:34] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-18 22:26:34] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Property AppBundle\Entity\Empresa::$ceIdMatriz does not exist" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\doctrine\common\lib\Doctrine\Common\Persistence\Mapping\RuntimeReflectionService.php line 80 {"exception":"[object] (ReflectionException(code: 0): Property AppBundle\\Entity\\Empresa::$ceIdMatriz does not exist at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\doctrine\\common\\lib\\Doctrine\\Common\\Persistence\\Mapping\\RuntimeReflectionService.php:80)"} []
[2016-09-18 22:34:17] request.INFO: Matched route "{route}". {"route":"lote_index","route_parameters":{"_controller":"AppBundle\\Controller\\LoteController::indexAction","_route":"lote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/lote/","method":"GET"} []
[2016-09-18 22:34:17] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-18 22:34:17] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: require(): Failed opening required 'C:\jcvpalma\xampps\htdocs\validcode_dev\var\cache\prod/doctrine/orm/Proxies\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\jcvpalma\xampps\php\PEAR')" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\doctrine\common\lib\Doctrine\Common\Proxy\AbstractProxyFactory.php line 209 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: require(): Failed opening required 'C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod/doctrine/orm/Proxies\\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\\jcvpalma\\xampps\\php\\PEAR') at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\doctrine\\common\\lib\\Doctrine\\Common\\Proxy\\AbstractProxyFactory.php:209)"} []
[2016-09-18 22:39:20] request.INFO: Matched route "{route}". {"route":"lote_index","route_parameters":{"_controller":"AppBundle\\Controller\\LoteController::indexAction","_route":"lote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/lote/","method":"GET"} []
[2016-09-18 22:39:21] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-18 22:39:21] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: require(): Failed opening required 'C:\jcvpalma\xampps\htdocs\validcode_dev\var\cache\prod/doctrine/orm/Proxies\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\jcvpalma\xampps\php\PEAR')" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\doctrine\common\lib\Doctrine\Common\Proxy\AbstractProxyFactory.php line 209 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: require(): Failed opening required 'C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod/doctrine/orm/Proxies\\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\\jcvpalma\\xampps\\php\\PEAR') at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\doctrine\\common\\lib\\Doctrine\\Common\\Proxy\\AbstractProxyFactory.php:209)"} []
[2016-09-18 22:40:56] request.INFO: Matched route "{route}". {"route":"lote_index","route_parameters":{"_controller":"AppBundle\\Controller\\LoteController::indexAction","_route":"lote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/lote/","method":"GET"} []
[2016-09-18 22:40:56] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-18 22:40:57] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: require(): Failed opening required 'C:\jcvpalma\xampps\htdocs\validcode_dev\var\cache\prod/doctrine/orm/Proxies\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\jcvpalma\xampps\php\PEAR')" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\doctrine\common\lib\Doctrine\Common\Proxy\AbstractProxyFactory.php line 209 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: require(): Failed opening required 'C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod/doctrine/orm/Proxies\\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\\jcvpalma\\xampps\\php\\PEAR') at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\doctrine\\common\\lib\\Doctrine\\Common\\Proxy\\AbstractProxyFactory.php:209)"} []
[2016-09-18 22:43:00] request.INFO: Matched route "{route}". {"route":"lote_index","route_parameters":{"_controller":"AppBundle\\Controller\\LoteController::indexAction","_route":"lote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/lote/","method":"GET"} []
[2016-09-18 22:43:01] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-18 22:43:01] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: require(): Failed opening required 'C:\jcvpalma\xampps\htdocs\validcode_dev\var\cache\prod/doctrine/orm/Proxies\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\jcvpalma\xampps\php\PEAR')" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\doctrine\common\lib\Doctrine\Common\Proxy\AbstractProxyFactory.php line 209 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: require(): Failed opening required 'C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod/doctrine/orm/Proxies\\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\\jcvpalma\\xampps\\php\\PEAR') at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\doctrine\\common\\lib\\Doctrine\\Common\\Proxy\\AbstractProxyFactory.php:209)"} []
[2016-09-18 22:45:13] request.INFO: Matched route "{route}". {"route":"lote_index","route_parameters":{"_controller":"AppBundle\\Controller\\LoteController::indexAction","_route":"lote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/lote/","method":"GET"} []
[2016-09-18 22:45:13] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-18 22:45:14] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: require(): Failed opening required 'C:\jcvpalma\xampps\htdocs\validcode_dev\var\cache\prod/doctrine/orm/Proxies\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\jcvpalma\xampps\php\PEAR')" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\doctrine\common\lib\Doctrine\Common\Proxy\AbstractProxyFactory.php line 209 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: require(): Failed opening required 'C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod/doctrine/orm/Proxies\\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\\jcvpalma\\xampps\\php\\PEAR') at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\doctrine\\common\\lib\\Doctrine\\Common\\Proxy\\AbstractProxyFactory.php:209)"} []
[2016-09-18 22:46:47] request.INFO: Matched route "{route}". {"route":"lote_index","route_parameters":{"_controller":"AppBundle\\Controller\\LoteController::indexAction","_route":"lote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/lote/","method":"GET"} []
[2016-09-18 22:46:47] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-18 22:46:48] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: require(): Failed opening required 'C:\jcvpalma\xampps\htdocs\validcode_dev\var\cache\prod/doctrine/orm/Proxies\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\jcvpalma\xampps\php\PEAR')" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\doctrine\common\lib\Doctrine\Common\Proxy\AbstractProxyFactory.php line 209 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: require(): Failed opening required 'C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod/doctrine/orm/Proxies\\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\\jcvpalma\\xampps\\php\\PEAR') at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\doctrine\\common\\lib\\Doctrine\\Common\\Proxy\\AbstractProxyFactory.php:209)"} []
[2016-09-18 22:46:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app_dev.phplote/"" at C:\jcvpalma\xampps\htdocs\validcode_dev\var\cache\prod\classes.php line 3167 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app_dev.phplote/\" at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod\\classes.php:3167, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod\\appProdUrlMatcher.php:634)"} []
[2016-09-18 22:47:26] request.INFO: Matched route "{route}". {"route":"itenslote_index","route_parameters":{"_controller":"AppBundle\\Controller\\ItensLoteController::indexAction","_route":"itenslote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/itenslote/","method":"GET"} []
[2016-09-18 22:47:26] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-18 22:47:26] request.CRITICAL: Uncaught PHP Exception Doctrine\ORM\Mapping\MappingException: "The target-entity AppBundle\Entity\Usuario cannot be found in 'AppBundle\Entity\ItensLote#ceUsuario'." at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\doctrine\orm\lib\Doctrine\ORM\Mapping\MappingException.php line 762 {"exception":"[object] (Doctrine\\ORM\\Mapping\\MappingException(code: 0): The target-entity AppBundle\\Entity\\Usuario cannot be found in 'AppBundle\\Entity\\ItensLote#ceUsuario'. at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\doctrine\\orm\\lib\\Doctrine\\ORM\\Mapping\\MappingException.php:762)"} []
[2016-09-18 22:47:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /apitenslote/"" at C:\jcvpalma\xampps\htdocs\validcode_dev\var\cache\prod\classes.php line 3167 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /apitenslote/\" at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod\\classes.php:3167, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod\\appProdUrlMatcher.php:634)"} []
[2016-09-18 22:50:53] request.INFO: Matched route "{route}". {"route":"lote_index","route_parameters":{"_controller":"AppBundle\\Controller\\LoteController::indexAction","_route":"lote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/lote/","method":"GET"} []
[2016-09-18 22:50:53] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-18 22:50:53] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: require(): Failed opening required 'C:\jcvpalma\xampps\htdocs\validcode_dev\var\cache\prod/doctrine/orm/Proxies\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\jcvpalma\xampps\php\PEAR')" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\doctrine\common\lib\Doctrine\Common\Proxy\AbstractProxyFactory.php line 209 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: require(): Failed opening required 'C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod/doctrine/orm/Proxies\\__CG__AppBundleEntityEmpresa.php' (include_path='.;C:\\jcvpalma\\xampps\\php\\PEAR') at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\doctrine\\common\\lib\\Doctrine\\Common\\Proxy\\AbstractProxyFactory.php:209)"} []
[2016-09-18 23:00:13] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /arquivolote" (from "http://localhost:8080/validcode_dev/web/app_dev.php/lote/")" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /arquivolote\" (from \"http://localhost:8080/validcode_dev/web/app_dev.php/lote/\") at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod\\appProdUrlMatcher.php:634)"} []
[2016-09-18 23:06:52] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /importacaolote/idlote/1" (from "http://localhost:8080/validcode_dev/web/app_dev.php/lote/")" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /importacaolote/idlote/1\" (from \"http://localhost:8080/validcode_dev/web/app_dev.php/lote/\") at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod\\appProdUrlMatcher.php:634)"} []
[2016-09-18 23:07:18] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /importacaolote/1/idlote" (from "http://localhost:8080/validcode_dev/web/app_dev.php/lote/")" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /importacaolote/1/idlote\" (from \"http://localhost:8080/validcode_dev/web/app_dev.php/lote/\") at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod\\appProdUrlMatcher.php:634)"} []
[2016-09-18 23:11:31] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /importacaolote/1/idlote"" at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /importacaolote/1/idlote\" at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod\\appProdUrlMatcher.php:634)"} []
[2016-09-19 02:09:47] request.INFO: Matched route "{route}". {"route":"importacaolote_index","route_parameters":{"_controller":"AppBundle\\Controller\\ArquivoLoteController::indexAction","_route":"importacaolote_index"},"request_uri":"http://localhost:8080/validcode_dev/web/importacaolote/","method":"GET"} []
[2016-09-19 02:09:48] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-19 02:09:48] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Catchable Fatal Error: Object of class DateTime could not be converted to string") in "arquivolote/index.html.twig" at line 20." at C:\jcvpalma\xampps\htdocs\validcode_dev\var\cache\prod\classes.php line 1645 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Catchable Fatal Error: Object of class DateTime could not be converted to string\") in \"arquivolote/index.html.twig\" at line 20. at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod\\classes.php:1645, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Object of class DateTime could not be converted to string at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\var\\cache\\prod\\twig\\fd\\fd55ed1f05ebe73223b95c6a521f25d0e15c5fbb2a0733873c8f94325ef389fc.php:56)"} []
[2016-09-19 05:16:05] request.INFO: Matched route "{route}". {"route":"importacaolote_edit","route_parameters":{"_controller":"AppBundle\\Controller\\ArquivoLoteController::editAction","id":"1","_route":"importacaolote_edit"},"request_uri":"http://localhost:8080/validcode_dev/web/importacaolote/1/edit","method":"GET"} []
[2016-09-19 05:16:05] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-19 05:16:06] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Form\Exception\LogicException: "The form's view data is expected to be an instance of class Symfony\Component\HttpFoundation\File\File, but is a(n) resource. You can avoid this error by setting the "data_class" option to null or by adding a view transformer that transforms a(n) resource to an instance of Symfony\Component\HttpFoundation\File\File." at C:\jcvpalma\xampps\htdocs\validcode_dev\vendor\symfony\symfony\src\Symfony\Component\Form\Form.php line 363 {"exception":"[object] (Symfony\\Component\\Form\\Exception\\LogicException(code: 0): The form's view data is expected to be an instance of class Symfony\\Component\\HttpFoundation\\File\\File, but is a(n) resource. You can avoid this error by setting the \"data_class\" option to null or by adding a view transformer that transforms a(n) resource to an instance of Symfony\\Component\\HttpFoundation\\File\\File. at C:\\jcvpalma\\xampps\\htdocs\\validcode_dev\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Form\\Form.php:363)"} []
